### DISCUSSION TITLE
manually adding features or capture markers

### DISCUSSION THREAD

**Discussion by albmef** [alicevision/Meshroom#1884] [Q&A] [answered]
Hi 
Anyone has some ideas of how to improve a scene capture by addind features by hand, editing 2D images or sticking markers on the subject ?
imagine i need to capture an object with shiny and soft curved surfaces (like most of plastic made objects) and keep exact mechanical dimension in order to use the mesh as a basis for some added mechanical feature, or just for repairing missing parts of this object. 
what could be the best way to obtain a strong matching of best key points ? editing 2D images ? adding SIFT or AKAZE features by hand ? sticking markers ? drawing dots on the object ? all of this methods ?
feel free to share your experiences, even the bad ones :)
best regards,
A

**Comment by natowi** [comment, marked as answer]
In the issues you can find dataset related questions and answers: https://github.com/alicevision/Meshroom/issues?q=+label%3Adataset-issues+

Here you can find tutorials on capturing images for photogrammetry and how to handle shiny objects: https://github.com/alicevision/Meshroom/wiki/Tutorials (https://www.3dscanstore.com/blog/3d-scanning-reflective-objects)

https://github.com/alicevision/Meshroom/wiki/Reconstruction-parameters
https://github.com/alicevision/Meshroom/wiki/Projected-Light-Patterns (polarization part)
https://github.com/alicevision/Meshroom/wiki/CCTAG-scaling
